usersync/usersyncers/syncer.go

```diff
@@ -1,6 +1,7 @@
 package usersyncers
 
 import (
+	"github.com/prebid/prebid-server/adapters/operaads"
```

Comment:
Can you please move this line down to keep imports in alphabetical order?

Reply:
fix,please check,thx.